website/pages/docs/plugins/sources/typeform/overview.mdx

```diff
@@ -0,0 +1,48 @@
+---
+name: Typeform
+stage: Preview
+title: Typeform Source Plugin
+description: CloudQuery Typeform source plugin documentation
+---
+# Typeform Source Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+import Configuration from "./_configuration.mdx";
+import Authentication from "./_authentication.mdx";
+
+<Badge text={"Latest: " + getLatestVersion("source", `typeform`)}/>
+
+The CloudQuery Typeform plugin pulls data from [Typeform](https://www.typeform.com/) and loads it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](/docs/plugins/destinations/overview)).
```

Comment:
Do the new python plugins work with all versions of the CLI (that support the latest protocol) or is there a minimum supported version?

Reply:
There is a minimum supported CLI version of v3.12.0, because it requires the `docker` registry. I've added this to the text, thanks for pointing it out 👍